3-terrarium/2-intro-to-css/README.md

```diff
@@ -20,6 +20,7 @@ In this lesson, we're going to add styles to our online terrarium and learn more
 You should have the HTML for your terrarium built and ready to be styled.
 
 ### Task
+This is how you link you're CSS `style.css` file to you're webpage.<br>
 
```

Comment:
We don't need this explanation here, also please beware of spelling errors, thanks